google-cloud-vision/lib/google/cloud/vision/annotation.rb

```diff
@@ -476,6 +476,8 @@ def web?
           !web.nil?
         end
 
+        # rubocop:disable Naming/MemoizedInstanceVariableName
+
```

Comment:
Should we just change the variable name for this one, rather than disabling the cop?